WowPacketParser/SQL/QueryBuilder.cs

```diff
@@ -62,6 +62,15 @@ public string Build()
             }
             else
             {
+                if (_conditions.Count > 100)
```

Comment:
better let it work directly if bigger than 1

Reply:
```suggestion
                if (_conditions.Count > 1)
```